src/tests/unit/tests/scanner/__snapshots__/get-rule-inclusions.test.ts.snap

```diff
@@ -16,9 +16,18 @@ exports[`getRuleInclusions getRuleInclusions matches snapshotted list of product
     "reason": "best practice rule that was investigated with no known false positives, implemented as an automated check.",
     "status": "included",
   },
+  "aria-braille-equivalent": {
```

Comment:
Enabling new rules usually includes adding documentation to info-examples. Have we done that for these?

Reply:
PM team is working on the documentation for these. We will release both Web extension and docs in sync. @nang4ally to add more details if required.